taxonomies/ingredients.txt

```diff
@@ -51163,2182 +51228,2054 @@ ciqual_food_code:en:53200
-# pnb:ہرا پیاز
-# pt-br:alho poró
-# sco:leek
-#smn:purjosiipuul
-#sms:purjoluukk
-# vls:pret
-#wuu:韭葱
-#xmf:პურასქია
-# yue:京蔥
-#plant: wikidata:en:Q21870138
-wikidata:en:Q1807269
-wikipedia:en:https://en.wikipedia.org/wiki/Leek
-carbon_footprint_fr_foodges_ingredient:fr:Poireau
-carbon_footprint_fr_foodges_value:fr:0.3
-ciqual_food_code:en:20039
-ciqual_food_name:en:Leek, raw
-ciqual_food_name:fr:Poireau, cru
-# 6266 in 27 @2021-10-26
-
-<en:leek
-en:concentrated leek juice
-da:koncentreret porresaft
-de:Konzentrierter Porreesaft
-fr:jus de poireau concentré
-#20 in 3 @2021-10-26
-
-<en:leek
-en:leek powder
-de:Lauchpulver
-#15 in 2 @2021-10-26
-
-# description:en:Asclepias is a genus of herbaceous, perennial, flowering plants known as milkweeds.
-<en:root vegetable
-en:asclepias
-sr:miluk
-wikidata:en:Q162153
-wikipedia:en:https://en.wikipedia.org/wiki/Asclepias
-
-<en:leek
-en:leek sprouts
-pl:kiełki pora
-
+<en:tuber
+en:yam
+de:Yam, Yamswurzel, Yamswurzeln
+fr:igname
+nutriscore_fruits_vegetables_nuts:en:no
+wikidata:en:Q8047551
+wikipedia:en:https://en.wikipedia.org/wiki/Yam_(vegetable)
+eurocode_2_group_3:en: 8.34.40
 
-# description:en:The ONION (Allium cepa L.), also known as the bulb onion or common onion, is a vegetable that is the most widely cultivated species of the genus Allium. They are usually served cooked, as a vegetable or part of a prepared savoury dish, but can also be eaten raw or used to make pickles or chutneys.
+# description:en:The POTATO is a starchy, tuberous crop from the perennial nightshade Solanum tuberosum.
 
-# <en:class:en:Vegetables
-# <en:class:en:Onions and their products
 <en:root vegetable
-en:onion, onions
-ar:بصل
-bg:лук, кромид лук
-bn:পিঁয়াজ
-ca:ceba
-cs:cibule
-cy:Nionyn
-da:løg
-de:Zwiebel, Zwiebeln
-el:κρεμμύδι, κρεμμύδια
-eo:cepo
-es:cebolla, Cebollas
-et:sibul
-eu:tipula
-fi:sipuli, sipulit
-fr:oignon, oignons
-he:בצל
-hr:luk, crni luk
-hu:hagyma, hagymák, vöröshagyma
-io:Onyono
-is:laukur
-it:cipolla, Cipolle
-ja:タマネギ, オニオン, たまねぎ, 玉葱, 玉ねぎ
-lt:svogūnai
-lv:sīpoli
-nb:løk
-nl:ui, uien, ajuin
-no:løk
-nv:tłʼohchin
-pl:cebula, cebuli, cebulowy, cebulowa, cebulowe, cebulka, cebulki
-pt:cebola
-ro:ceapă, ceapa
-ru:лук, лук репчатый
-sv:lök
-tr:soğan
-uk:Цибуля
-vi:hành tây
-zh:洋葱
-# aeb-arab:راس بصل
-# ckb:پیاز
-wikidata:en:Q3406628
-wikipedia:en:https://en.wikipedia.org/wiki/Onion
-# ingredient/onion has 12754 products in 18 languages @2018-12-06
-# category/onions has 233 products @2019-05-22
-carbon_footprint_fr_foodges_ingredient:fr:Oignons
-carbon_footprint_fr_foodges_value:fr:0.6
-ciqual_food_code:en:20034
-ciqual_food_name:en:Onion, raw
-ciqual_food_name:fr:Oignon, cru
-
-<en:onion
-en:organic onion
-hr:bio luk ljubičasti
-
-# description:en:Cebula dymka is a name for young onion meant to be pickled or consumed raw
-
-<en:onion
-pl:cebula dymka
-
-<en:onion
-fr:oignon semoule
-
-<en:onion
-fr:oignons grillés en poudre
-
-<en:onion
-fr:oignon fumé en poudre
-
-<en:onion
-en:onion purée
```

Comment:
-> pl (pasta)

Reply:
pasta is a translation of en:paste, but the whole section is commented out for some reason